liferay-echo/liferay-portal
# LPS-181478 Remove FF

https://issues.liferay.com/browse/LPS-181478

# Motivation
Removes the FF for the Structure Tree migration to the Clay TreeView and fixes some bugs founded while removing the FF.
[Link to task](https://issues.liferay.com/browse/LPS-151678)
[Link to subtask](https://issues.liferay.com/browse/LPS-181478)

# Proposed Solution
Remove the FF flag and the old code and styles. We have also fixed some bugs with the calculation of the expanded nodes and some styles that were not being applied correctly for mapped nodes.

## Steps to Verify:

1. Go to home page and click on edit
2. In the left sidebar click on Browser section
3. Open the tree nodes and check that all the actions work for each node (rename, hide/show, save composition duplicate, delete, drag and drop)
4. Use a page with a master page and check that everything works fine
